workshop_app/urls.py

```diff
@@ -17,7 +17,7 @@
 from workshop_app import views
 
 js_info_dict = {
-    'packages': ('recurrence', ),
+    'packages': ('recurrence',),
```

Comment:
Update this PR with a git pull --rebase, this should not be here as I have already merged the recurrence removal PR